core/src/main/java/io/undertow/protocols/alpn/JDK9AlpnProvider.java

```diff
@@ -64,6 +61,8 @@ public JDK9ALPNMethods run() {
                     // methods would exist and could result in issues. These methods can still be used by providing the
                     // io.undertow.protocols.alpn.jdk8=true system property and support for Java 8 known in the
                     // SSLEngine implementation being provided.
+                    final String value = System.getProperty(JDK8_SUPPORT_PROPERTY);
```

Comment:
Easier to use Boolean.getBoolean(String propertyName) instead.

Reply:
Sorry, just realized you want to default to true. Please ignore my comment.

Reply:
That wouldn't work because we want to default to `true` so if it's not defined, e.g. `null`, we need it to be true.